qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -90,6 +112,108 @@ def quantum_instance(
         else:
             self._quantum_instance = quantum_instance
 
+    @property
+    def user_parameters(self) -> Union[ParameterVector, Sequence[Parameter]]:
+        """Return the vector of user parameters."""
+        return self._user_parameters
+
+    @user_parameters.setter
+    def user_parameters(self, user_params: Union[ParameterVector, Sequence[Parameter]]) -> None:
```

Comment:
The typehints here should inlcude Optional I think since the getter will return None if user parameters are never set, and the setter allows you to pass in None presumably to remove any user parameters and let them all be controlled by the kernel.

Reply:
Ok, but I think the getter above should be Optional for the return type since it will return None by default when there are no user parameters right. And not allowing the setter to take None I guess means that you cannot go back to having no user parameters once you chose to have some - aside from setting a new feature map that has the 'side-effect' of removing them.